### PR TITLE
lnrpc: display node and peer features via rpc

### PR DESCRIPTION
**Pull request by cfromknecht**
This PR exposes features in three new places on over rpc:
 * `GetNodeInfo`: node features
 * `DescribeGraph`: node features
 * `ListPeers`: remote init features

In addition, all rpc feature vectors are now displayed as maps instead of lists since this is more useful when interacting with the rpc responses and closer to how they are represented internally.

### CODE REVIEW

File: rpcserver.go [hunk 1]
```diff
@@ -4137,13 +4138,16 @@ func (r *rpcServer) GetNodeInfo(ctx context.Context,
 		nodeAddrs = append(nodeAddrs, nodeAddr)
 	}
 
+	features := invoicesrpc.CreateRPCFeatures(node.Features)
```
Comment (carlaKC) [inline rpcserver.go]: Can inline this in `NodeInfo`?

File: lnrpc/rpc.proto [hunk 2]
```diff
@@ -2069,6 +2069,7 @@ message LightningNode {
     string alias = 3 [ json_name = "alias" ];
     repeated NodeAddress addresses = 4 [ json_name = "addresses" ];
     string color = 5 [ json_name = "color" ];
+    repeated Feature features = 6 [ json_name = "features" ];
```
Comment (carlaKC) [inline lnrpc/rpc.proto]: nit: we have `[json_name` (without a space) more than `[ json_name` (with space) in `rpc.proto`, not sure of which is correct, just noticed it.

Comment (cfromknecht) [reply lnrpc/rpc.proto]: yeah lol both styles are used throughout the file, i just tried to match the surrounding definitions

File: rpcserver.go (hunk 1 above)
Comment (cfromknecht) [reply rpcserver.go]: ends up being 1 char too long :/

**Pull request merged**